README.md

```diff
@@ -6,38 +6,55 @@ platform and CoralBlox model. This project also contains the Reef Guide app.
 
 ## Angular Developer Setup
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 18.0.6.
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli)
+version 18.0.6.
 
 **One-time setup:**
-1. Install [Node v22](https://nodejs.org/en/download/package-manager)  
```

Comment:
trailing while space in Markdown is meaningful. It forces a line break. Looks like backslash is an alternative and probably better since it's more explicit.

Reply:
whether you like the line break here or not is up to you.

Reply:
Oh sorry I missed this. Yes a line break is needed here. Usually I just add a preceding space to align with the dot points text. That _usually_ works